### ISSUE TITLE
When using a API token, the login is not correctly checked

### ISSUE THREAD

**Issue by kifeo** [p4r4n0y1ng/pyfgt#2]
In the case you use the API token method, the login() says always Login successful even if the token is not correct.
Then with the get(), then the 401 HTTP error code (not authorized) raises an exception at line 298 of fortigate.py with a JSON decode error, as the 401 error code is not JSON formatted

**Comment by p4r4n0y1ng** [comment]
Yes......there really is no "login" for using the API token. The only reason login() is called is so both "processes" - username or token - follows the same line through the code. Login will ALWAYS pass if you're using token. You should be able to use one of the included Exception objects to grab it, but it's exactly the same as if the login had failed. I'll check to see if it's getting caught by the FGTBaseException if utilized. In any case - if you check out the docs, there's no "login" concept with the token - so this is just a "pass" concept until a real call is made.  Do you have some other thoughts that I might add?

**Comment by kifeo** [comment]
I would say that the login could check a small fortigate command to be able to verify that the token authorization is correct ?

**Comment by p4r4n0y1ng** [comment]
Hmmmmm.....man this sat sorry - I didn't see your input here. So, you want me to catch a bad "key" basically and throw back a login failure by doing something like just checking for a 200 from a get sys status type concept?
